pkg/sql/catalog/catprivilege/fix.go

```diff
@@ -138,3 +138,54 @@ func MaybeFixPrivileges(
 	}
 	return changed
 }
+
+// MaybeUpdateGrantOptions iterates over the users of the descriptor and checks
+// if they have the GRANT privilege - if so, then set the user's grant option
+// bits equal to the privilege bits
+func MaybeUpdateGrantOptions(ptr **descpb.PrivilegeDescriptor) bool {
+	p := *ptr
+	changed := false
+	if p.Version >= descpb.GrantOptionVersion { // do not apply changes
```

Comment:
nit: period and specify why we do not need to apply changes (on >= `GrantOptionVersion` means grant bits are set)